<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>658</width>
    <height>361</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Аналитическая иерархия</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="mainTab">
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab1">
       <attribute name="title">
        <string>Таблица попарных сравнений</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="ComparisionTable" name="CompTable1Lvl" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab2">
       <attribute name="title">
        <string>Таблица попарных сравнений 2 уровня</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QTabWidget" name="tab2Lvl">
          <property name="tabPosition">
           <enum>QTabWidget::South</enum>
          </property>
          <property name="tabShape">
           <enum>QTabWidget::Triangular</enum>
          </property>
          <property name="currentIndex">
           <number>1</number>
          </property>
          <widget class="QWidget" name="tab">
           <attribute name="title">
            <string>Tab 1</string>
           </attribute>
          </widget>
          <widget class="QWidget" name="tab_2">
           <attribute name="title">
            <string>Tab 2</string>
           </attribute>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab3">
       <attribute name="title">
        <string>Таблица попарных сравнений 3 уровня</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QTabWidget" name="tab3Lvl">
          <property name="tabPosition">
           <enum>QTabWidget::South</enum>
          </property>
          <property name="tabShape">
           <enum>QTabWidget::Triangular</enum>
          </property>
          <property name="currentIndex">
           <number>1</number>
          </property>
          <widget class="QWidget" name="tab_3">
           <attribute name="title">
            <string>Tab 1</string>
           </attribute>
          </widget>
          <widget class="QWidget" name="tab_4">
           <attribute name="title">
            <string>Tab 2</string>
           </attribute>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab4">
       <attribute name="title">
        <string>Таблица глобальных критериев</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <widget class="GlobalCriteriesTable" name="glCriteriesTable"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab5">
       <attribute name="title">
        <string>Таблица результатов</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="0" column="0">
         <widget class="ResultTable" name="resTable"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>658</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;Файл</string>
    </property>
    <addaction name="acNew"/>
    <addaction name="acLoad"/>
    <addaction name="acClose"/>
    <addaction name="separator"/>
    <addaction name="acSave"/>
    <addaction name="acSaveAs"/>
    <addaction name="separator"/>
    <addaction name="acExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Справка</string>
    </property>
    <addaction name="acAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="acNew">
   <property name="text">
    <string>Новый</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="acLoad">
   <property name="text">
    <string>Загрузить</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="acSave">
   <property name="text">
    <string>Сохранить</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="acSaveAs">
   <property name="text">
    <string>Сохранить как...</string>
   </property>
  </action>
  <action name="acExit">
   <property name="text">
    <string>Выход</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </action>
  <action name="acAbout">
   <property name="text">
    <string>О программе...</string>
   </property>
  </action>
  <action name="acClose">
   <property name="text">
    <string>&amp;Закрыть</string>
   </property>
  </action>
  <action name="acAddTab">
   <property name="text">
    <string>addTab</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ComparisionTable</class>
   <extends>QWidget</extends>
   <header>comparisiontable.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GlobalCriteriesTable</class>
   <extends>QTableWidget</extends>
   <header>globalcriteriestable.h</header>
  </customwidget>
  <customwidget>
   <class>ResultTable</class>
   <extends>QTableWidget</extends>
   <header>resulttable.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
